.github/workflows/unit-tests.yml

```
@@ -58,7 +58,7 @@ jobs:
       - uses: actions/checkout@v4
       - uses: maxim-lobanov/setup-xcode@v1
         with:
-          xcode-version: ${{ inputs.xcode-version || 'latest-stable' }}
+          xcode-version: ${{ inputs.xcode-version || '26.0.1' }}
```

I think this was mixed up with ios version?
```suggestion
          xcode-version: ${{ inputs.xcode-version || '26.0' }}
```